packages/rest/README.md

```diff
@@ -12,18 +12,17 @@ with:
 - new custom routing engine (special thanks to @bajtos)!
 - tools for defining your application routes
 - OpenAPI 2.0 spec (`swagger.json`/`swagger.yaml`) generation
-- a default sequence implementation to manage the request and response lifecycle
+- a default sequence implementation to manage the request and response life cycle
+
 ## Installation
-To use this package, you'll need to install both `@loopback/core` and
-`@loopback/rest`.
+To use this package, you'll need to install `@loopback/rest`.
 
 ```shell
-$ npm install --save @loopback/core @loopback/rest
+$ npm i @loopback/rest
 ```
 
 ## Basic Use
-Here's a basic "Hello World" application using `@loopback/core` and
-`@loopback/rest`:
+Here's a basic "Hello World" application using `@loopback/rest`:
 
   ```ts
   import {RestApplication, RestServer} from '@loopback/rest';
```

Comment:
super minor nitpick: extract out `rest.getSync('rest.port')` since that's what we do in the CLI generated template